client/src/pages/Profile.js

```diff
@@ -103,7 +103,7 @@ const URLS = {
   website: [websiteIcon],
 };
 
-const getHref = (url) => (url.startsWith("http") ? url : `//${url}`);
+const getHref = url => (url.startsWith("http") ? url : `//${url}`);
```

Comment:
Why all these format changes? It seems like you are using a different linter or prettier config.

Reply:
Yes, during development I have used prettier standard javascript format, I did not noticed before pushing the code. So, I mentioned @rabbi08 in the earlier message. Sorry about that, hereafter I will make sure about it.